pkg/cli/cli.go

```diff
@@ -641,14 +641,15 @@ const (
 
 func runCreateCVDCommand(c *cobra.Command, args []string, flags *CreateCVDFlags, opts *subCommandOpts) error {
 	if len(args) > 0 {
-		// Load and parse the passed environment configuration.
-		data, err := os.ReadFile(args[0])
+		// Load and parse the passed canonical configuration.
```

Comment:
environment is the right word, canonical are the ones that you get by just importing @phone or @tablet inside those files.

Reply:
I see, changed it to `environment specification` as discussed offline.